doc/whats_new/v1.1.rst

```diff
@@ -150,6 +150,12 @@ Changelog
   error message when setting invalid hyper-parameters with `set_params`.
   :pr:`21542` by :user:`Olivier Grisel <ogrisel>`.
 
+- |Enhancement| All scikit-learn estimators now save the sklearn version
```

Comment:
This did not make it into v1.1, which means this change log entry needs to move to v1.2.